medusa/subtitle_providers/legendastv.py

```diff
@@ -246,7 +249,7 @@ def search_titles(self, title):
                     if match:
                         title['season'] = int(match.group('season'))
                     else:
-                        logger.warning('No season detected for title %d', title_id)
+                        logger.debug('No season detected for title id: %d and name: %s', title_id, name)
```

Comment:
I only found all this issues in LTV because of the logs.
We discard/continue loop without logging. Its hard to debug without it.


IMO there is no point showing as warning as there is nothing to do about it. Why warn user about that? Debug is fine